build.gradle

```diff
@@ -15,8 +15,8 @@ app {
 
 dependencies {
     include "com.enonic.lib:lib-thymeleaf:2.0.1"
-    compile "com.enonic.xp:core-api:${xpVersion}"
-    compile "com.enonic.xp:portal-api:${xpVersion}"
+    include "com.enonic.xp:core-api:${xpVersion}"
```

Comment:
Check docs, looks like compile should be replaced with api or implementation https://docs.gradle.org/current/userguide/upgrading_version_6.html#sec:configuration_removal